documentation/docs/release-notes/2.44.0.1.md

```diff
@@ -0,0 +1,75 @@
+# Percona Monitoring and Management 2.44.0-1
+
+**Release date**: February 10th, 2025                                                                            
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB. 
+
+It provides tools to observe database health, analyze performance trends, troubleshoot issues, and execute database management tasks—whether your databases are on-premises or in the cloud.
+
+
+## Security advisory: Critical vulnerability
+
+A critical vulnerability has been identified in OVA installations of PMM 2.38 and above, as described in [CVE-xxxx-xxxx](https://cve.mitre.org/cgi-bin/cvename.cgi?name=xxxx-xxxx). If you are running any of these OVA installations, we strongly recommend upgrading immediately to this release.
+
+The vulnerability was caused by default service account credentials used in OVA provisioning, which had weak password security and enabled SSH login access. 
+These issues could allow unauthorized SSH access and potential privilege escalation to `root` via `sudo` capabilities.
+
+### Scope of impact
+- Affected: PMM Open Virtual Appliance (OVA) installations, version 2.38 and above
+- Not affected: PMM Docker installations, PMM Amazon Machine Images (AMIs)
+- Severity: CRITICAL when the SSH port is exposed to the Internet
+
+### Fix and security enhancements 
+
+This release strengthens security in OVA deployments by removing unnecessary system accounts during the initial setup. The new OVA installations automatically remove unnecessary system accounts and include built-in security measures:
+
+- automatic security hardening during the first boot
+
+- enhanced system access controls to prevent unauthorized access
+
+- improved default security configurations for safer deployments
+
+### Upgrade instructions
+
+To secure your system, follow these steps to upgrade:
+
+1. Download and deploy the new OVA from [Percona website](https://www.percona.com/downloads).
+2. Stop services on your current installation: 
+    ```sh 
+    supervisorctl stop all
+    ```
+3. Back up and transfer your data:
+    ```sh
+    cd /srv
+    tar -cvf srv.tar .
+    # Transfer to new server
+    ```
+
+4. Deploy data on the new installation:
+    ```sh
+    supervisorctl stop all
+    cd /srv
+    tar -xvf srv.tar
+    supervisorctl start all
+    ```
+5. Update DNS records or swap IP addresses to ensure uninterrupted Client connectivity.
+
+### Verification steps
+
+After upgrading, verify that your system is functioning correctly:
+
+1. Check service status to confirm both PMM Client and PMM Server are running.
```

Comment:
I think we should either provide commands here or point to the UI. Is it `supervisorctl status` or something else?